device/mellanox/x86_64-mlnx_msn3700-r0/ACS-MSN3700/port_config.ini

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/port_config.ini
```

Comment:
what am i missing? i expect to see new independed port_config.ini for sn3700 not the one used for sn2700

Reply:
@liat-grozovik 
This file is the same for both MSN2700 and MSN3700 (both has the same amount of interfaces) thanks to the SAI team, who provided a unified logic for interfaces port mapping (SAI HW lanes and HW Spec lanes is not the same).